### PR TITLE
Installation

### LINKED ISSUE #1: Add installation instructions
We can add installation instructions and a requirements file for installing jupyter notebook, nilearn, and other needed dependencies. We can also add binder links. I can work on this.

### PR DESCRIPTION
**Pull request by ymzayek**
Closes #1

I added a binder link and local installation instructions.
For binder, I would still need to test if the notebooks will run without crashing as I am unaware of their memory requirements.

### CODE REVIEW

**Comment by ymzayek** [comment]
All the current notebooks run fine in binder. Just need to shut down the kernel after finishing working with each notebook to free up memory for the next one.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+Install the requirements (this will install nilearn and its dependencies, as well as Jupyter-notebooks):
+
+```
+$ pip install -r requirements.txt
+```
+
+Launch the notebooks:
+
+```
+$ cd Notebooks
```
Comment (bthirion) [inline README.md]:
```suggestion
$ cd notebooks
```

**Review by bthirion — COMMENTED** [review]
There seems to be a configuration problem under binder too.

**Comment by ymzayek** [comment]
@bthirion what configuration issue do you see with binder? It runs fine for me but stalls when I test it with the requirements file on my fork (though it worked yesterday). Another option is to have it quickly start without (pre)installing any requirements. And then manually install them with pip from jupyterlab's terminal. This is way faster and will be fairly easy to follow by the participants as it's just one line: `pip install nilearn matplotlib`.

**Comment by bthirion** [comment]
When I launch binder from https://github.com/ymzayek/fMRI_workshop/tree/installation
I get the following error:
ModuleNotFoundError: No module named 'matplotlib'

**Comment by ymzayek** [comment]
Hmm like before JupyterLab even starts? 
Or when running a notebook? Because this won't work yet until the PR is merged because it's running the repo from `fMRI_workshop/main` and the requirements file isn't included yet.
Maybe let's take a look together.

**Comment by bthirion** [comment]
Yes, it is when running a notebook. Then you can probably ignore my comment.
Best,

**Comment by ymzayek** [comment]
Ok to merge this?

**Pull request merged**